CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 2.2.1
+ - fix: Uses valid Dart import paths for endpoints on all platforms.
```

Comment:
Suggestion: Change to: `fix: fixes an issue where invalid Dart import paths would be generated on Windows.`

Since this is both regarding the endpoint import in project files and the protocol and endpoint import in the test tools.